cmd/storagenode/main.go

```diff
@@ -89,6 +95,15 @@ func cmdSetup(cmd *cobra.Command, args []string) (err error) {
 }
 
 func main() {
+	// Figure out the executable name.
+	exe, err := os.Executable()
+	if err != nil {
+		exe = "storagenode.exe"
+	}
+	// Make the windows graphical launch text a bit more friendly.
+	cobra.MousetrapHelpText = fmt.Sprintf("This is a command line tool.\n\n"+
+		"This needs to be run from a Command Prompt.\n"+
+		"Try running \"%s help\" for more information", exe)
```

Comment:
if you put this cobra.MousetrapHelpText business in our process.Exec function you would only need to have it in one spot instead of in all of these places